go.mod

```diff
@@ -39,3 +39,5 @@ require (
 	knative.dev/networking v0.0.0-20210304153916-f813b5904943
 	knative.dev/pkg v0.0.0-20210309024624-0f8d8de5949d
 )
+
+replace knative.dev/eventing => github.com/cardil/eventing v0.11.1-0.20210525135852-116524fc7b9f
```

Comment:
/hold

Until https://github.com/openshift/knative-eventing/pull/1284 is merged.

Reply:
/unhold

The openshift/knative-eventing#1284 is merged now.